src/sentry/api/endpoints/group_hashes_split.py

```diff
@@ -211,123 +203,182 @@ def _unsplit_group(group: Group, hash: str, hierarchical_hashes: Optional[Sequen
 
 def _get_group_filters(group: Group):
     return [
-        Condition(Column("timestamp"), Op.GTE, group.first_seen),
-        Condition(Column("timestamp"), Op.LT, group.last_seen + datetime.timedelta(seconds=1)),
         Condition(Column("project_id"), Op.EQ, group.project_id),
         Condition(Column("group_id"), Op.EQ, group.id),
+        Condition(Column("timestamp"), Op.GTE, group.first_seen),
```

Comment:
Why is it beneficial to keep one timestamp condition but remove the other?

Reply:
I just reverted this change as evan pointed out that this won't work and added some comments